website/config/routes.js

```diff
@@ -499,6 +499,7 @@ module.exports.routes = {
   'GET /fleetctl-preview':   '/try-fleet/fleetctl-preview',
   'GET /try-fleet/sandbox-expired':   '/try-fleet/fleetctl-preview',
   'GET /try-fleet/sandbox':   '/try-fleet/fleetctl-preview',
+  'GET /docs/using-fleet/mdm-macos-updates': '/learn-more-about/os-updates',
```

Comment:
```suggestion
  'GET /docs/using-fleet/mdm-macos-updates': '/docs/using-fleet/mdm-os-updates',
```

Reply:
See comment here: https://github.com/fleetdm/fleet/pull/16270#discussion_r1471431182

Reply:
Ah, I see. "learn more" link is only for use in core product. Similar reasoning to here: https://github.com/fleetdm/fleet/pull/16270#discussion_r1470634094

Reply:
✅